src/js/Calculator.js

```diff
@@ -0,0 +1,133 @@
+export default class Calculator {
+  _operation;
+  _expression;
+  _limit;
+
+  constructor() {
+    this._operation = '';
+    this._expression = '0';
+    this._limit = 3;
+  }
+
+  get expression() {
+    return this._expression;
+  }
+  resetOperation = () => {
+    this._operation = '';
+  };
+  resetExpression = () => {
+    this._expression = '0';
+  };
+  resetDigitLimit = () => {
+    this._limit = 3;
+  };
+  useDigitLimit = () => {
+    this._limit = this._limit - 1;
+  };
+
+  addExpression = (value) => {
+    this._expression = this._expression + value;
+  };
+  checkExpression = () => {
+    if (this._expression === 'Infinity' || this._expression === 'NaN') {
+      this.resetExpression();
+    }
+  };
+
+  handleAC = () => {
+    this.resetDigitLimit();
+    this.resetOperation();
+    this.resetExpression();
+  };
+
+  handleDigit = (digit) => {
+    this.checkExpression();
+    if (this._limit === 0) {
+      alert('숫자는 세자리까지만 입력 가능합니다!');
+      return;
+    } else if (this._expression === '0') {
+      this._expression = digit.toString();
+    } else {
+      this.addExpression(digit);
+    }
```

Comment:
else 체이닝은 앞단의 조건을 계속해서 인지한 상태로 다음 조건을 읽어야해서 가독성에 좋지 않다고 생각해요.

return을 통해 else를 제거해보면 어떨까요?

```suggestion
    if (this._limit === 0) {
      alert('숫자는 세자리까지만 입력 가능합니다!');
      return;
    } 
    
    if (this._expression === '0') {
      this._expression = digit.toString();
      return
    } 
    
     this.addExpression(digit);
    
```